assets/sass/components/key-metrics/_googlesitekit-chip-tab-group.scss

```diff
@@ -68,19 +68,48 @@
 						display: none;
 					}
 				}
+
+				.mdc-tab__content {
+					position: initial;
+				}
 			}
 
 			/** Mobile version style adaptation - end **/
 		}
 
 		.googlesitekit-chip-tab-group__chip-item {
 			@extend %chip-item;
+			overflow: visible;
 
 			svg {
 				margin-right: 7px;
 			}
 		}
 
+		.googlesitekit-chip-tab-group__chip-item-new-dot {
+			align-items: center;
+			background-color: #fff;
+			border-radius: 50%;
+			display: flex;
+			height: 10px;
+			justify-content: center;
+			position: absolute;
+			right: 0;
+			top: 0;
+			width: 10px;
+
+			&::after {
+				background-color: $c-violet-v-500;
+				border-radius: 50%;
+				content: " ";
```

Comment:
Why the whitespace in this content? 🤔 

```suggestion
				content: "";
```

Reply:
Old habit, there was issue long time ago where pseudo element might not show in some browsers without that extra space, but yeah not needed any more you are correct

Reply:
I was thinking that too, takes me way back 😆